src/content/docs/integrations/prometheus-integrations/install-configure/remote-write-drop-data.mdx

```diff
@@ -0,0 +1,96 @@
+Let's say one of your targets is sending a bunch of extra attributes you're not interested in receiving. These might include things like high cardinality attributes such as unique machine identifiers, JVM IDs, or similar. In this case, you need to change both the `remote_write` and the `scrape_configs` section of the YAML file. The result will look something like this. 
+
+```
+remote_write:
+- url: https://metric-api.newrelic.com/prometheus/v1/write?prometheus_server=macbook-server-cluster
+  bearer_token: <redacted>
+  write_relabel_configs:
+    - regex: 'extraLabelToRemove.*'
+      action: 'labeldrop'
+...
+scrape_configs:
+  # The job name is added as a label `job=<job_name>` to any timeseries scraped from this config.
+  - job_name: 'node'
+    # Override the global default and scrape targets from this job every 5 seconds.
+    scrape_interval: 5s
+    static_configs:
+      - targets: ['localhost:9100']
+        labels:
+          group: 'production'
+          keepLabelName1: 'please-keep-me'
+          extraLabelToRemove: 'please-remove-me'
+          extraLabelToRemove1: 'please-remove-me'
+          extraLabelToRemove2: 'please-remove-me'
+          extraLabelToRemove4: 'please-remove-me'
+          extraLabelToRemove3: 'please-remove-me'
+          extraLabelToRemove5: 'please-remove-me'
+```
+
+## Prometheus or NerdGraph? [#prometheus-vs-nerdgraph]
+
+There are advantages to both dropping data using the method described on this page and using using [NerdGraph](/docs/accounts/accounts/data-management/drop-data-using-nerdgraph/). This section is intended to help you figure out which method is better for your specific needs and preferences. 
+
+### Considerations for the Prometheus config file method
+
+With this method, your dropped data never leaves the associated Prometheus instance. This is a valuable feature if bytes transferred is a cost consideration on the app hosting side.
+
+However, this method may be less appealing than the [NerdGraph](/docs/accounts/accounts/data-management/drop-data-using-nerdgraph/)option due to the following considerations: 
+- Maintained via config yaml files that need to be loaded onto each Prometheus instance (or via a shared storage mechanism) 
+- Requires access to Prometheus server, meaning that either:
+- - The server needs to be restarted
+- - Served must be be accessed at port with path `/-/reload` (assuming the server has lifecycle management enabled as described here in the Prometheus [configuration](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#configuration) docs. 
+
+### Considerations for NerdGraph data droping
+
+NerdGraph is a great option if you want to manage all your data dropping in a single place. It can also be updated easily via the API, and requires no restart or interaction with Prometheus. However, this method applies rules to all incoming data points. This means that you should set up your rules with careful consideration using `WHERE` filtering. 
```

Comment:
Line 87: The comma in the second sentence started with "It can also..." is not needed.

Reply:
good catch!